### PR TITLE
Adding more supported file extensions

### PR DESCRIPTION
**Pull request by holloway**
I had a content directory with 30k user supplied images with various file extensions, and some of them were .JPG etc which are supported by imagemagick. This one liner adds support for them.

### CODE REVIEW

**Comment by firetix** [comment]
Thank you @holloway

File: index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = function(options) {
             return cb(new gutil.PluginError('gulp-image-optimization', 'Streaming not supported'));
         }
 
-        if (['.jpg', '.jpeg', '.png', '.gif'].indexOf(path.extname(file.path)) === -1) {
+        if (['.jpg', '.JPG', '.JPEG', '.jpeg', '.png', '.PNG', '.gif', '.GIF'].indexOf(path.extname(file.path)) === -1) {
```
Comment (firetix) [inline index.js]: Do we have to use the lowercase and uppercase?

Comment (holloway) [inline index.js]: Yes, on case-sensitive file systems (like most Linux filesystems).

Alternatively the code could be written as .indexOf(path.extname(file.path.toLowerCase()) and then just have lower case extensions, but as (I presume) we don't want to match paths ending in eg ".JpEg" then I thought a list of capitalised extensions was a simpler change.

Comment (firetix) [inline index.js]: I'm going to put the extension to lowercase before comparing thanks for the feedback! I'll get this done this weekend. 

Cheers!

**Pull request merged**